npm/src/compiler/compiler.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 import { IDiagnostic } from "../../lib/node/qsc_wasm.cjs";
+import { PackageType } from "../../lib/web/qsc_wasm.js";
```

Comment:
Can you not just add the extra import on the line above? The node and web packages should be the same (we use the node one as it has a little less noise in it).

Reply:
When I try to make that change, I start getting failures when compiling the playground that look like this:
```
Running esbuild
X [ERROR] Could not resolve "util"

    ../npm/lib/node/qsc_wasm.cjs:4:45:
      4 │ const { TextDecoder, TextEncoder } = require(`util`);
        ╵                                              ~~~~~~

  The package "util" wasn't found on the file system but is built into node. 
  Are you trying to bundle for node? You can use "platform: 'node'" to do    
  that, which will remove this error.
```
If I leave it as importing from the .js instead of adding to the .cjs, then it works. I'm hesitant to change any platform settings for how esbuild runs just to try and address this error, so I think I'm better off leaving this as a separate import (though it's possible this is an issue with my local configuration, I suppose).